cli/popper/commands/cmd_search.py

```diff
@@ -0,0 +1,222 @@
+import click
+import os
+import glob
+import popper.utils as pu
+from io import BytesIO
+from popper.cli import pass_context
+import yaml
+import requests
+import json
+
+
+@click.command(
+    'search',
+    short_help='Used to search for an experiment in your pipeline folder')
+@click.argument('pipeline_name', required=False)
+@click.option(
+    '--skip-update',
+    help=('Skip updating the metadata and search on the local cache'),
+    is_flag=True
+)
+@pass_context
+def cli(ctx, pipeline_name, skip_update):
+    """Searches for the specified pipelines inside the  GitHub
+    organizations specified in the .popper.yml file.
+
+    The organization at -> https://github.com/popperized is present
+    by default.
+
+    If no pipeline name is specified by the user, then a list of all the
+    pipelines from all the listed organizations and their respective
+    repositories will be returned.
+
+    Example:
+
+    popper search quiho
+
+    would result in:
+
+    popperized/quiho-popper
+
+    """
+
+    project_root = pu.get_project_root()
+
+    empty_query = False
+
+    if not pipeline_name:  # checks if the query is empty or not
+        empty_query = True
+
+    POPPER_GITHUB_API = ""
```

Comment:
can we rename it to `POPPER_GITHUB_API_TOKEN`? That would be more closer to the "standard" practice (e.g. what `homebrew` and other cli tools do)